src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -352,7 +352,7 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
         }
         if (_.isArray(cellValue)) {
           cellValue = cellValue.join(' ');
-        } else if (_.isNumber(cellValue)) {
+        } else if (_.isNumber(cellValue) || _.isBoolean(cellValue)) {
```

Comment:
Please extend the tests to capture this change :)

Reply:
Could you please point me to an example where you checked the other datatypes?

Reply:
Oh sorry just saw your comment what I mend was something like a search test that searches for a boolean. Could you be so kind and add a search test for a boolean?